packages/angular-output-target/__tests__/output-angular.spec.ts

```diff
@@ -33,14 +42,21 @@ import { Components } from 'component-library';
       directivesProxyFile: '../component-library-angular/src/proxies.ts',
     };
 
-    const finalText = generateProxies(components, pkgData, outputTarget, rootDir);
+    const finalText = generateProxies(
+      components,
+      pkgData,
+      outputTarget,
+      rootDir,
+      config,
+      {} as any,
+    );
     expect(finalText).toEqual(
       `/* tslint:disable */
 /* auto-generated angular directive proxies */
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, EventEmitter, NgZone } from '@angular/core';
 import { ProxyCmp, proxyOutputs } from './angular-component-lib/utils';
 
-import { Components } from '../../angular-output-target/dist/types/components';
+import { Components } from '../component-library-angular/dist/types/components';
```

Comment:
This change is worth reviewing. I haven't made a library that doesn't include the component package's name.

Reply:
The updated value is actually the correct value - the original test was running from the root of  the angular-output-target path, this one now takes into account the root of the Stencil project (preferred)